lineman/spec-e2e/invitations_spec.coffee

```diff
@@ -12,12 +12,19 @@ describe 'Invitations', ->
       invitationsHelper.openInvitationsModal()
       expect(invitationsHelper.groupDropdown().isPresent()).toBe(true)
 
-    it 'successfully opens a model for a specific group', ->
+    it 'successfully opens a modal for a specific group', ->
       groupsHelper.load()
       invitationsHelper.openInvitationsModal()
       expect(invitationsHelper.groupDropdown().getText()).toContain('Dirty')
       expect(invitationsHelper.invitableInput().isPresent()).toBe(true)
 
+    it 'lets the inviter provide a custom message', ->
```

Comment:
Again, I think it's overkill to have a whole test just for this simple functionality; if you wanted to write a test which added a custom message and checked that the email we sent out contained it, that'd be a good e2e, otherwise I'd just take this out.